tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -447,10 +447,6 @@ File and sync features in a shared scenario
 
 - [apiShareManagementToShares/mergeShare.feature:105](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareManagementToShares/mergeShare.feature#L105)
 
-#### [not possible to move file into a received folder](https://github.com/owncloud/ocis/issues/764)
-
-- [apiShareManagementBasicToShares/createShareToSharesFolder.feature:500](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareManagementBasicToShares/createShareToSharesFolder.feature#L500)
```

Comment:
already done in https://github.com/owncloud/ocis/pull/4434/files#diff-3a67c616b0ac8adead10fbf21d7f2aff96dc8b7aa883b97a24b0dc865fde1e41